plugins/zoom/icon_zoom/tasks/monitor_sign_in_out_activity/task.py

```diff
@@ -99,16 +100,16 @@ def loop(self, state: Dict[str, Any], custom_config: Dict[str, Any]):  # noqa: C
             )
             self.logger.info(f"A custom start time of {last_day} will be used")
         else:
-            last_day = self._format_datetime_for_zoom(dt=self._get_datetime_last_24_hours())
+            last_day = self._format_datetime_for_zoom(dt=self._get_datetime_last_x_hours(hours=cutoff))
```

Comment:
should this variable name be updated? `start_time` or something now that we can't guarantee it's the last day?

Reply:
this has been updated to use start time as the var